lib/livebook/config.ex

```diff
@@ -194,6 +194,13 @@ defmodule Livebook.Config do
     @feature_flags[key]
   end
 
+  def allowed_uri_schemes() do
+    case schemes = Application.fetch_env!(:livebook, :allowed_uri_schemes) do
+      [] -> ""
+      _ -> String.to_charlist(schemes)
+    end
```

Comment:
```suggestion
    Application.fetch_env!(:livebook, :allowed_uri_schemes)
```

Please also add a doc and spec as above :)